crawler.py

```diff
@@ -124,6 +131,27 @@ def visit_title(*args, **kargs):
                         (self._fix_url(line.strip(), ""), 0))
         except IOError:
             pass
+    
+    # Returns the inverted index
+    def get_inverted_index(self):
+        return self.inverted_index
+
+    # Generates the resolved index everytime the function is called
+    def get_resolved_index(self):
+        resolvedIndex = dict()
+        for keys in self.inverted_index:
```

Comment:
could perhaps use `for word_id in self.inverted_index:`. will be fairly self-explanatory that way and wouldn't need comments